### DISCUSSION TITLE
How to add extra padding to the top of the Y-axis

### DISCUSSION THREAD

**Discussion by ariefzuhri** [patrykandpatrick/vico#448] [Ideas]
I have a column chart with data labels, as shown in the screenshot below.
![Screenshot 2023-10-23 083841](https://github.com/patrykandpatrick/vico/assets/54942127/6b6e8ad0-c9ea-43c0-9340-c18d8ae6b99c)

I expected the data labels to always be in a fixed position at the top of the bars. However, in reality, if there isn't enough space for the data labels, they end up overlapping with the bars.

I don't want to use trickery, such as increasing the maximum value of the Y-axis. Instead, I'd like to add extra padding between the chart and its container. This way, when the bars reach the maximum value of the Y-axis, the data labels can still have space to be displayed at the top of the bars without overlapping.

Is there a way to achieve this?

**Comment by Jacob3075** [comment]
were you able to find a solution to this?

**Reply by ariefzuhri** [reply]
Thus far, this chart library doesn't offer a clean way to achieve this. Consequently, I had to resort to increasing the maximum value of the Y-axis.

However, in my case, the Y-axis represents hours on a clock, limited to a maximum value of 24. When a data point approaches or equals 24, I need to increase the maximum value beyond 24. This approach doesn't align with the accurate representation of time.

**Reply by patrickmichalik** [reply]
Please note that you can use `AxisItemPlacer.Vertical` to hide the extra labels and lines, @ariefzuhri, so 24 is the greatest value displayed.

**Reply by ariefzuhri** [reply]
I'll give it a try later, thanks!

**Reply by patrickmichalik** [reply]
Sure thing.

**Comment by patrickmichalik** [comment]
Hello! This isn’t possible yet, but we’ll be looking into adding such an option. Thanks for the suggestion!

**Reply by ariefzuhri** [reply]
Thank you for considering that! Looking forward to seeing the potential addition in the future 😃
